cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -12,9 +12,6 @@
 {% if pillar['allocate_node_cidrs'] is defined -%}
   {% set allocate_node_cidrs = "--allocate-node-cidrs=" + pillar['allocate_node_cidrs'] -%}
 {% endif -%}
-{% if pillar['enable_experimental_api'] is defined -%}
-  {% set enable_experimental = "--enable-experimental=" + pillar['enable_experimental_api'] -%}
-{% endif -%}
 
```

Comment:
Line 4 is definitely dead code but pillar definition is still used by apiserver.manifest I believe